bin/civihr-install

```diff
@@ -0,0 +1,820 @@
+#!/bin/bash
```

Comment:
I haven't used extension here because it needs to be executable as a command and as I've read executable files should be without any extension (did some research  :https://google.github.io/styleguide/shell.xml#File_Extensions ). Also I think using sh command to execute a .sh file won't be much user friendly.